styles.css

```diff
@@ -16,4 +17,14 @@
     text-align: center;
     color: white;
     border-top-right-radius: 1rem;
+}
+
+.side-bar{
+    border:1px solid black;
+    flex-basis:15rem;
+} 
+
+.radio-btn{
+    height:20px;
```

Comment:
you can use relative unit like rem instead of px

Reply:
Understood, thanks